add-ons/FAIRFEM/EFEM_SSD_MBvsMF.m

```diff
@@ -0,0 +1,38 @@
+% ==================================================================================
+% (c) Lars Ruthotto 2012/07/26, see FAIR.2 and FAIRcopyright.m.
```

Comment:
can you please update the description of the examples and use the new header from the other examples? We decided to let Github keep track of the authorship for us. 

Please add a brief description about what this example is supposed to do.